### DISCUSSION TITLE
how can i use csq

### DISCUSSION THREAD

**Discussion by WithSJ** [CsqLang/Csq3#6] [Q&A] [answered]
i directly run ./csq but these shows me.
Enter 3 arguments: <name> <current-dir> <compiler-path>

  what is 3 arguments and  why it's need.

**Comment by WithSJ** [comment]
After Reading code i understand that 
i need to create hello.csq ".csq" file 
first arg is name of file than its path than compiler path

**Comment by AniketKumar2500** [comment, marked as answer]
no actually firstly you need to provide name of file than the current
directory and at last directory in which compiler is present.
